problem19.rb

```diff
@@ -0,0 +1,2 @@
+require 'date'
+puts Date.new(1901,1,1).upto(Date.new(2000,12,31)).find_all { |d| d.mday == 1 && d.wday == 0 }.count
```

Comment:
```suggestion
puts (Date.new(1901,1,1)..Date.new(2000,12,31)).find_all { |d| d.mday == 1).select(&:sunday?).count
```
Это просто как вариант, твоя версия тоже очень хорошая